packages/near-membrane-base/src/environment.ts

```diff
@@ -44,23 +44,23 @@ export class VirtualEnvironment {
 
     private readonly blueGetSelectedTarget: GetSelectedTarget;
 
-    private readonly blueGetTransferableValue: GetTransferableValue;
+    private blueGetTransferableValue: GetTransferableValue;
```

Comment:
```suggestion
    private readonly blueGetTransferableValue: GetTransferableValue;
```

Reply:
all these readonly private fields should continue to be readonly... they are not used anywhere else after the last set of changes, so restoring them to the original state.